tests/run-make/avr-rjmp-offset/rmake.rs

```diff
@@ -0,0 +1,49 @@
+//! Regression test for #129301/llvm-project#106722 within `rustc`.
+//!
+//! Some LLVM-versions had wrong offsets in the local labels, causing the first
+//! loop instruction to be missed. This test therefore contains a simple loop
+//! with trivial instructions in it, to see, where the label is placed.
+//!
+//! This must be a `rmake`-test and cannot be a `tests/assembly`-test, since the
+//! wrong output is only produced with direct assembly generation, but not when
+//! "emit-asm" is used, as described in the issue description of #129301:
+//! https://github.com/rust-lang/rust/issues/129301#issue-2475070770
+use run_make_support::{llvm_objdump, rustc};
```

Comment:
Hm, this probably needs some flags like

```
//@ only-avr
//@ needs-llvm-components: avr (otherwise will not work if local llvm built without avr component)
```

Reply:
Oh yes, this is missing. I'll add the "needs-llvm-components"-directive. The `only-avr` doesn't seem to work, though: if I add this line, the test always gets ignored.